department-of-veterans-affairs/vets-website
# Arf 84629 Form 21a step 3

## Summary

- Added step pages to align with the 21A V2 Designs flow (Note: there are some aspects that align with V3 Designs that were confirmed with design as mostly set - if you do not see the UI element in V2 check the V3 tab.)
- [Designs](https://www.figma.com/design/2afIGOMII0uRI5ck1dWo1w/Form-21a---Apply-for-Accreditation-(CA-%26-Attorneys)?node-id=262-21987&t=JXSWf514L6nfbf8n-4)
- Added e2e testing
- Accredited Representative Facing Team

## Related issue(s)

- [Merge Form 21a Step 3 (Military History)](https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/83871)
- [[ARF] Digital 21a](https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/83627)

## Testing done

- Navigate from Introduction page through all 7 steps especially step 2 to review and submit
- Ensure the pages align with [designs](https://www.figma.com/design/2afIGOMII0uRI5ck1dWo1w/Form-21a---Apply-for-Accreditation-(CA-%26-Attorneys)?node-id=262-21987&t=JXSWf514L6nfbf8n-4)

## Screenshots

### Before - these pages did not exist

### After
<img width="924" alt="Screenshot 2024-06-14 at 10 44 03 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/100dad5b-7ed6-4134-91f6-f1ce9c1abb87">
<img width="917" alt="Screenshot 2024-06-14 at 10 44 21 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/fd387e64-7c0a-4ed8-981f-b3105d623df9">
<img width="930" alt="Screenshot 2024-06-14 at 10 44 48 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/ac66facf-ce71-42d0-8b7a-06d59f4c2765">
<img width="916" alt="Screenshot 2024-06-14 at 10 45 32 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/27e5827b-4971-4c9b-a268-8bb916f13d3a">
<img width="931" alt="Screenshot 2024-06-14 at 10 45 42 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/22990386/045b5a31-dfcb-4040-96cf-ea85c3460bbc">

## Acceptance criteria

### Quality Assurance & Testing
